### ISSUE TITLE
Remote SSH tunnel support

### ISSUE THREAD

**Issue by Morethanevil** [eduardolat/pgbackweb#38]
The webinterface is awesome, the S3 support too. I wanted to know if SSH-tunnel support is planned? I have 3 different servers, one homeserver at home, a VPS and a dedicated server. All of them are running Postgres, but I only want to have pgbackweb at home on my LAN. If I want to back up the other 2 servers, I would need to expose pgbackweb to the web, which I don't want. A VPN is not an option in my case. A remote connection would be cool in this case. Pgbackweb could connect via SSH to the other hosts and then perform its actions

**Comment by AiAutomationAgency** [comment]
Supporting SSH for backs instead if S3 would be create.

**Comment by codedoga** [comment]
Also mentioned here: https://github.com/eduardolat/pgbackweb/issues/5

**Comment by eduardolat** [comment]
@Morethanevil Of course, the project is alive; it's simply stable.

The main problem is that I, as the maintainer, am currently going through an extremely busy phase of my life. I'm easily working on my projects 12 hours a day, 7 days a week, and right now it's impossible for me to move forward with this project, which isn't really generating any benefits for me (apart from the Postgres backups, of course).

However, I plan to continue developing it as soon as I have a little more free time and improve the project as much as I can, adding many of the features requested in the issues and other things I have planned, such as MySQL, Mongo, SQLite backups, etc.

I hope you can understand that I don't have the time right now, but I will continue working on this (unless I die), and i'm very sorry to hear that you're no longer using the project. I really did create it with a lot of love and without expecting anything in return. But I understand that I haven't dedicated much time to it lately. I hope that can change soon.

**Comment by Morethanevil** [comment]
Sorry to hear that you are so busy, but look from the other side: a web application, that not gets updates within months is a security risk. The Go library and its dependencies got several updates too.

**Comment by eduardolat** [comment]
You're right, I'll try to at least keep the dependencies up to date while I get time to move forward with the new features.
